src/components/Footer/StyledFooter.js

```diff
@@ -12,8 +12,10 @@ const ContactPanelRow = styled(Row)``
 
 // Individual social media contact column
 const ContactPanelColumn = styled(Col)`
-  justify-content: center;
-  padding: 30px;
+${genericStyles}
+  justify-content: ${props => (props.justify ? props.justify : 'center')};
+  text-align: ${props => (props.align ? props.align : 'center')};
```

Comment:
These two lines above should be done like `${props => (!props.align && 'text-align: center;')};`